docs/measurements/procedures/rigid_waist_shift.md

```diff
@@ -11,11 +11,71 @@
+The knob setting is defined as follows, with regards to Beam1 (reverse for Beam2):
+
+- Unit setting of +$1$: shifts the waist to the _left_ of the IP ($s_{waist}^{B1} < s_{IP}^{B1}$)
+- Unit setting of -$1$: shifts the waist to the _right_ of the IP ($s_{waist}^{B1} > s_{IP}^{B1}$)
+
+### The Knobs in LSA
+
+Since the triplet knob for a unit setting of +$1$ is the exact opposite magnet powering of the knob for a unit setting of -$1$, only a single triplet knob per IP has been added to LSA, which can then be trimmed with a factor of $\pm1$.
+On the other hand, the re-matching quadrupole knobs are specific to a given direction of the shift, so two of them per IP (one per direction) were added to LSA, with the *pos* and *neg* suffixes.
+These should always be trimmed with a factor of $1$.
+
+As a consequence, respect the following rules when applying the knobs below:
+
+ - When trimming the triplet knob with a factor of +$1$, use the *pos* re-matching quadrupoles knob.
+ - When trimming the triplet knob with a factor of -$1$, use the *neg* re-matching quadrupoles knob.
+
+??? info "LSA Triplets Knobs"
+
+    |  IP   |                        Knob Name                         |
+    | :---: | :------------------------------------------------------: |
+    |  IP1  | LHCBEAM/MD_ATS_2020-05_04_BX_RigidWaistShift_Triplet_IP1 |
+    |  IP5  | LHCBEAM/MD_ATS_2020-05_04_BX_RigidWaistShift_Triplet_IP5 |
+
+??? info "LSA Independent Quadrupoles Knobs"
+
+    |  IP   |  Beam  |                      Knob Name                      |
+    | :---: | :----: | :-------------------------------------------------: |
+    |  IP1  | Beam 1 | LHCBEAM/MD_ATS_2022_05_04_B1_RigidWaitsShift_IP1neg |
+    |       |        | LHCBEAM/MD_ATS_2022_05_04_B1_RigidWaitsShift_IP1pos |
+    |       | Beam 2 | LHCBEAM/MD_ATS_2022_05_04_B2_RigidWaitsShift_IP1neg |
+    |       |        | LHCBEAM/MD_ATS_2022_05_04_B2_RigidWaitsShift_IP1pos |
+    |  IP5  | Beam 1 | LHCBEAM/MD_ATS_2022_05_04_B1_RigidWaitsShift_IP5neg |
+    |       |        | LHCBEAM/MD_ATS_2022_05_04_B1_RigidWaitsShift_IP5pos |
+    |       | Beam 2 | LHCBEAM/MD_ATS_2022_05_04_B2_RigidWaitsShift_IP5neg |
+    |       |        | LHCBEAM/MD_ATS_2022_05_04_B2_RigidWaitsShift_IP5pos |
+
+??? example "An Example"
+    One wishes to implement the waist shift at IP1, shifting the waist to the left of IP (for Beam1).
+    To do so they need to apply the *LHCBEAM/MD_ATS_2020-05_04_BX_RigidWaistShift_Triplet_IP1* with a factor of +$1$.
+    Since they used a +$1$ factor on the triplet knob, the corresponding re-matching quadrupoles knobs to use are those with the *pos* suffix, for Beam1 and Beam2: *LHCBEAM/MD_ATS_2022_05_04_B2_RigidWaitsShift_IP1pos* and *LHCBEAM/MD_ATS_2022_05_04_B2_RigidWaitsShift_IP1pos*.
```

Comment:
* To do so we
*Since we used 

*are those with the *pos* suffix and need to be set to +$1$.
Note that in the case of a negative RWS shift, the *neg* quadrupole rematching knobs need to be set to +$1$.

I'd be as explicit as possible with the signs of the knobs, those usually cause a lot of confusion, more so during night shifts.

Reply:
Additional clarifications in [8928d02](https://github.com/pylhc/pylhc.github.io/pull/74/commits/8928d02b9557cbabaaf5a5a0f68fa2641da23430)